extensions/VisualEditor/wikia/modules/ve/ui/pages/ve.ui.WikiaParameterPage.js

```diff
@@ -6,6 +6,107 @@
  */
 
 /**
+ * Restores infobox dialog info when going back from media dialog
+ *
+ * @param {ve.ui.WindowManager} win
+ * @param {ve.dm.WikiaTransclusionModel} model - original tranclusion model of infobox dialog
+ * @param {string} imageName - image name from media dialog
+ * @param {string} paramName - Name of a field (e.g. image1)
+ */
+function restoreInfoboxDialog( win, model, imageName, paramName ) {
```

Comment:
You're right, we don't want to leak these functions. Encapsulating them in objects would be PITA but fortunately there is another convetion in VE where they just used IIFE, e.g. in https://github.com/Wikia/app/blob/c0047a035f31d435cc307c1a87e6d88e61fbe5f9/extensions/VisualEditor/lib/ve/src/dm/ve.dm.ModelRegistry.js#L6

That's what I did in https://github.com/Wikia/app/pull/12110/commits/628c9f45e86b2c709440250aea40f0c541a245f0